### ISSUE TITLE
Keyboard Accessibility issue with sub menus in main menu

### ISSUE THREAD

**Issue by AlexGStapleton** [siteorigin/vantage#136]
You're unable to tab through the submenus in the main navigation. This is a major accessibility issue for those physically unable to use a mouse.

**Comment by Misplon** [comment]
@adiraomj Want to tackle with #223 at the same time? Thanks dude :)
